### ISSUE TITLE
Focus is not visible on Scrolling cards under Trusted By section: A11y_ONNX Runtime & Ecosystem_Runtime_Focus Visible

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/onnxruntime#24996]
#A11yMAS; #A11ySev2; #MAS2.4.7; #FocusVisible; #DesktopWeb; #Chrome; #Win11; #ONNX Runtime & Ecosystem; #Benchmark; #rev:moda; #BM-HCL-ONNX Runtime&Ecosystem-Jun2025; #HCL;
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**
### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Chrome Version: 120.0.6099.130
OS: Windows 11 version 23H2

### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page appeared.
3. Tab till "Toggle Scrolling" button and press enter.
4. Tab till the scrolling cards.
5. Observe that Focus is not visible on the cards.
###Actual:
Focus is not visible on the scrolling cards under the Trusted By section.
**Note: Env is inconsistent some times issue is not repro**
### Expected:
Focus should be visible on the scrolling cards under the Trusted By section.
### User Impact:
keyboard users and low vision users will face difficulties if focus is not visible on the cards.
### Recommendations:
Refer[ Developer Resources (sharepoint.com)]
### MAS Reference:
[MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)
 Check out   [MAS 2.4.7: Focus Visible Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-2.4.7-Focus-Visible-Guide.aspx?)
Attachment

[A11y_ONNX Runtime & Ecosystem_Runtime_Focus Visible 2.zip](https://github.com/user-attachments/files/20654734/A11y_ONNX.Runtime.Ecosystem_Runtime_Focus.Visible.2.zip)

**Comment by eserscor** [comment]
The icons now get focused when hitting tab

**Comment by eserscor** [comment]
Discussed with Accessibility Test Team and they have confirmed that the issue has been fixed and can be closed
